argocd-notifications/base/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: argocd
+
+resources:
+- 'https://raw.githubusercontent.com/argoproj-labs/argocd-notifications/v1.2.0/manifests/install.yaml'
+patchesStrategicMerge:
+- 'https://raw.githubusercontent.com/argoproj-labs/argocd-notifications/v1.2.0/catalog/install.yaml'
```

Comment:
do we need this catalog? It doesn't seem like we are using any of the triggers/templates added here.